syntax = "proto3";


package helloworld;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}

  rpc GetAge (GetAgeRequest) returns (GetAgeReply) {}

}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}

// The request message containing the user's age.
message GetAgeRequest {
  string age = 1;
}

// The response message containing the users age
message GetAgeReply {
  string message = 1;
}